Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -1571,11 +1571,13 @@ script:
       - PATCH
       - DELETE
       required: true
-    - description: the API path starting with forward slash (/).
+    - description: the API path (without a leading forward slash '/').
       name: path
       required: true
     - description: data to send in body, can be json.
       name: body
+    - description: the custom API root (like /api/tasks/).
+      name: custom_api_root
```

Comment:
please remove the root only custom_api is enough for the name (change accordingly in other places)